src/OpenFga.Sdk/Telemetry/Standard20TelemetryTypes.cs

```diff
@@ -0,0 +1,99 @@
+//
+// OpenFGA/.NET SDK for OpenFGA
+//
+// API version: 1.x
+// Website: https://openfga.dev
+// Documentation: https://openfga.dev/docs
+// Support: https://openfga.dev/community
+// License: [Apache-2.0](https://github.com/openfga/dotnet-sdk/blob/main/LICENSE)
+//
+// NOTE: This file was auto generated. DO NOT EDIT.
+//
+
+
+namespace OpenFga.Sdk.Telemetry {
```

Comment:
OTEL support is completely broken in this PR. Codepaths using OTEL types need to be wrapped in a compiler directive as it's unsupported under .NET Standard 2.0. This mocked out class makes no sense.